android/src/app/organicmaps/maplayer/MapButtonsController.java

```diff
@@ -242,8 +242,10 @@ private void applyMove(View frame, float translationY)
 
   public void updateButtonsVisibility()
   {
-    updateButtonsVisibility(mInnerLeftButtonsFrame.getTranslationY(), mInnerLeftButtonsFrame);
-    updateButtonsVisibility(mInnerRightButtonsFrame.getTranslationY(), mInnerRightButtonsFrame);
+    if(mInnerRightButtonsFrame != null && mInnerLeftButtonsFrame != null) {
```

Comment:
`mInnerRightButtonsFrame` could be null when `mInnerLeftButtonsFrame` is not (and also the opposite) depending on the current layout loaded.

So it would be better to check each UI frame before each associated function: only check `mInnerLeftButtonsFrame != null` before calling `updateButtonsVisibility(mInnerLeftButtonsFrame.getTranslationY...` and same for `mInnerRightButtonsFrame`.